src/dune_config_file/display.ml

```diff
@@ -40,7 +40,12 @@ let console_backend = function
        Dune_console.Backend.dumb
      | true ->
        (match Config.(get threaded_console) with
-        | `Enabled -> Dune_threaded_console.progress ()
+        | `Enabled ->
+          Dune_threaded_console.progress
+            ~refresh_rate:
+              (match Dune_util.Execution_env.inside_emacs with
```

Comment:
Kind of weird that being inside emacs overrides an explicit setting.

Reply:
Not really sure how else to handle it. I'm not overly fond of these emacs specific behaviours to begin with.

Reply:
How about this:

1. if the config is set, do not override it
2. if it's not set, default to 60 outside of emacs. 15 inside of emacs.

Reply:
Done.